Dockerfile

```diff
@@ -22,10 +22,16 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 # Cray Image Management Service image build environment Dockerfile
-FROM artifactory.algol60.net/csm-docker/stable/docker.io/opensuse/leap:15.4 as base
+# NOTE - currently the algol version does not have arm64 platform - update this when it does
+#FROM artifactory.algol60.net/csm-docker/stable/docker.io/opensuse/leap:15.4 as base
```

Comment:
Did you file a Jira or otherwise make a request to get this leap image imported into our Artifactory?

Reply:
CASMCMS-8590